### DISCUSSION TITLE
刚发现一个IOS客户端，不知道是否可靠

### DISCUSSION THREAD

**Discussion by SfYangYang** [AlistGo/alist#4515] [General]
不是之前论坛里发的那个。
![20230531_153640000_iOS](https://github.com/alist-org/alist/assets/130076124/97d74077-785f-4cd9-b1f7-8598ccf1ba05)

**Comment by ethanjtch** [comment]
1. 不知道是什么问题，我连不到本地的alist，只能连到https的（我用replit搭建测试）
2. 这个不是在iPhone部署Alist的，只能挂载，fireball/document等都可以
